api/src/main/java/org/openmrs/module/insuranceclaims/api/service/impl/InsuranceClaimServiceImpl.java

```diff
@@ -32,4 +40,9 @@ private void updateQuantityApproved(InsuranceClaim claimToUpdate, InsuranceClaim
             claimToUpdate.setApprovedTotal(totalBenefit);
         }
     }
+
+    @Override
+    public List<InsuranceClaim> getAllInsuranceClaims(Integer patientId) throws APIException {
```

Comment:
public methods should be first, all methods in this call should be in transaction